cmd/goldsmith/ui/ui.go

```diff
@@ -8,7 +8,35 @@ import (
 
 // FFTStreamer buffers a streamer and also computes an FFT whos chunks are available on [FFTChan].
 type FFTStreamer interface {
-	beep.Streamer
+	beep.Streamer // Are the methods from this being used? if not I'd leave it out
```

Comment:
Ah I think this is true!  I was thinkin all explicit interface like but go doesn't need me to tell it this and that interface overlap